components/footer/Footer.js

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
nice job reusing your Logo and Button components here!

think about how we can refactor this into two smaller components - the privacy policy and the chat bot, like so:
```
//Footer.js
const Footer = () => {
  return (
    <footer>
      <Disclaimer/>
      <HelpChat/>
    </footer>
  );
}
```
in this way when we add complex functionality specific to the chat bot section of the footer, it can be encapsulated in its own component.

Reply:
Okay makes sense - will work on bringing over finished chatbot component from previous build via codesandbox. Chatbox will have another child component - chatWindow (text to display once chatbox is clicked).

Reply:
@achang-tcs Chatbox is not completed - will create a separate branch / pr for this one - Thanks!